src/Controls/src/Core/Platform/Android/TabbedPageManager.cs

```diff
@@ -622,7 +625,42 @@ protected virtual ColorStateList GetItemIconTintColorState()
 			if (_newTabIconColors != null)
 				return _newTabIconColors;
 
-			int defaultColor = barItemColor.ToPlatform().ToArgb();
+			int defaultColor;
+			
+			if (barItemColor is not null)
+			{
+				defaultColor = barItemColor.ToPlatform().ToArgb();
+			}
+			else
+			{
+				var styledAttributes = 
+					TintTypedArray.ObtainStyledAttributes(_context.Context, null, Resource.Styleable.NavigationBarView, Resource.Attribute.bottomNavigationStyle, 0);
+
+				try
+				{
+					var defaultColors =  styledAttributes.GetColorStateList(Resource.Styleable.NavigationBarView_itemIconTint);
+					if (defaultColors is not null)
+					{
+						defaultColor = defaultColors.DefaultColor;		
+					}
+					else
+					{
+						// These are the defaults currently set inside android
+						// It's very unlikely we'll hit this path because the 
+						// NavigationBarView_itemIconTint should always resolve
+						// But just in case, we'll just hard code to some defaults
+						// instead of leaving the application in a broken state
+						if(IsDarkTheme)
+							defaultColor = new Color(1, 1, 1, 0.6f).ToPlatform();
```

Comment:
Instead of hardcoding, can we recover default values from resources?

Reply:
That's what the code above this is doing. This is fallback code if for some reason we fail to retrieve the defaults from the resources